docs/landing-zone-v2/onboarding-client.md

```diff
@@ -1,36 +1,97 @@
 # Client Onboarding
 
-## Add client folder(s) to the landing zone repository
+<!-- vscode-markdown-toc -->
+* [Introduction](#Introduction)
+* [Setup](#Setup)
+* [Add client-setup package](#Addclient-setuppackage)
+* [Add the client-landing-zone package](#Addtheclient-landing-zonepackage)
+* [Next Step](#NextStep)
 
-1. Move to the root of the landing zone
+<!-- vscode-markdown-toc-config
+	numbering=false
+	autoSave=true
+	/vscode-markdown-toc-config -->
+<!-- /vscode-markdown-toc -->
+
+## <a name='Introduction'></a>Introduction
+
+You will execute this procedure to provision the foundational resources in GCP for each client. These resources are required before you can create application project.
+
+## <a name='Setup'></a>Setup
+
+1. Move to the folder where the `gatekeeper-policies` and `core-landing-zone` packages are located
+
+    ```shell
+    cd <FOLDER>
+    ```
+
+1. Create the following folder structure for clients
+
+    ```text
+    ├── clients
+    │   ├── <client1-name>
+    │   ├── <client2-name>
+    │   ├── ...
+    ├── core-landing-zone
+    ├── gatekeeper-policies
+    ```
+
+## <a name='Addclient-setuppackage'></a>Add client-setup package
+
+1. Get the client-setup package
+
+> **!!! Update the command below with the proper VERSION, you can locate it in the package's CHANGELOG.md, for example, '0.0.1'. Use 'main' if not available**
+
```

Comment:
I am wondering about the advice to use 'main'. During version updates it was necessary to use a specific version for kpt pkg update to work on renamed target directories @stephanemillaire-ssc - lets discuss.

Reply:
@Michael-McAuley-SSC I'll wait for a come back from you two. not resolving the conversation for now

Reply:
Please advise using versions over main - obtained through the .release-please-manifest.json. Alternatively, each package changelog contains the history if there is a requirement to use an older version.

Reply:
Agreed with this. Main would be our equivalent of building from source and while that's possible it shouldn't be the first option and version should be preferred.

Reply:
latest commit include the new recommendation